css-three-column-layout/styles.css

```diff
@@ -0,0 +1,62 @@
+
+* {
+  background-color: #1996da;
+  font-family: Lato Arial, Helvetica, sans-serif;
+}
+
+.row {
+  display: flex;
+}
+
+.container {
+  max-width: 1200px;
+  margin: 0 auto;
+  color: white;
+  padding: 20px 40px;
+}
+
+.column-third {
+  width: calc(100% / 3);
+  padding: 25px;
+  text-align: center;
```

Comment:
Only `width` or `flex-basis` on `column` classes, you should make an additional rule for this.